src/deep-flat/deepFlat.spec.js

```diff
@@ -0,0 +1,16 @@
+//import { deepFlat } from './deep-flat.js';
```

Comment:
`import { deepFlat } from './deepFlat.js';`
also importing the deepFlat function from './deepFlat.js'  for testing purposes